### PR TITLE
Make it possible to submit day-of runs as "upcoming"

### PR DESCRIPTION
**Pull request by mcoughlin**
Closes https://github.com/skyportal/skyportal/issues/3249

### CODE REVIEW

File: static/js/components/AssignmentForm.jsx [hunk 1]
```diff
@@ -84,8 +84,10 @@ const AssignmentForm = ({ obj_id, observingRunList }) => {
   }));
   const classes = useStyles();
 
+  // the use of integer dates leads to some upcoming runs being
+  // left out depending on the timezone
   const upcomingObservingRuns = observingRunList.filter((run) =>
-    dayjs().isBefore(dayjs(run.run_end_utc))
+    dayjs().isBefore(dayjs(run.run_end_utc).add(2, "day"))
```
Comment (guynir42) [inline static/js/components/AssignmentForm.jsx:90]: Are you sure this doesn't accidentally add runs that are too far in the future?

**Review by guynir42 — COMMENTED** [review]
I have one thing I'm not sure about.

File: static/js/components/AssignmentForm.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/AssignmentForm.jsx:90]: Nope because of how the comparison is done. I checked and you get the day 'before' with this.

**Comment by mcoughlin** [comment]
@guynir42 you can merge it if you are happy.

**Review by guynir42 — APPROVED** [review]

**Pull request merged**